terraform/services/github-actions/deploy-roles/main.tf

```diff
@@ -0,0 +1,55 @@
+data "aws_iam_policy" "developer_boundary_policy" {
+  name = "developer-boundary-policy"
+}
+
+resource "aws_iam_policy" "runner" {
+  name = "github-actions-runner"
+
+  description = "Runner has access to everything within the permissions boundary to allow for terraform and other deploy steps"
+
+  inline_policy {
+    name = "my_inline_policy"
```

Comment:
The inline_policy section must be in the aws_iam_role resource. Also, let's use a more logical name like "all_within_boundary".